copy_with_style.js

```diff
@@ -175,33 +177,42 @@ class Copy_With_Style {
 		this.observe = observe;
 		this.show_progress = progress ? true : false;
 		this.copy_wrapper = copy_wrapper;
-		this.class_button = class_button,
-		this.class_preparing = class_preparing,
-		this.class_ready = class_ready,
-		this.deep_exclusions = deep_exclusions,
-		this.shallow_exclusions = shallow_exclusions,
-		this.extra_deep_exclusions = extra_deep_exclusions,
-		this.extra_shallow_exclusions = extra_shallow_exclusions,
-		this.debug = debug;
+		(this.class_button = class_button),
+			(this.class_preparing = class_preparing),
+			(this.class_ready = class_ready),
+			(this.deep_exclusions = deep_exclusions),
+			(this.shallow_exclusions = shallow_exclusions),
+			(this.extra_deep_exclusions = extra_deep_exclusions),
+			(this.extra_shallow_exclusions = extra_shallow_exclusions),
+			(this.debug = debug);
 		this.log_performance = log_performance;
 		this.log_HTML_to_console = log_HTML_to_console;
 		this.log_text_to_console = log_text_to_console;
 		this.check_clone_integrity = check_clone_integrity;
 		this.classes_to_debug = classes_to_debug;
 		this.styles_to_debug = styles_to_debug;
-		this.tags_to_debug = tags_to_debug.map(name => name.toUpperCase()); // tagNames reported in JS are always Uppercase
+		this.tags_to_debug = tags_to_debug.map((name) => name.toUpperCase()); // tagNames reported in JS are always Uppercase
```

Comment:
I don't much care one way or the other here but am curious and cautious about consistency, vs spot changes. I use the first syntax (without brackets) everywhere. Looks cleaner to me. But if it were to change to bracketed I'd look to do it across the whole file I guess.

Reply:
The file is formatted by prettier once I did not realize while editing, and the casing should be constant after this action.